mltrace/cli/cli.py

```diff
@@ -274,16 +294,27 @@ def flag(output_id: str, address: str = ""):
 
 
 @mltrace.command("unflag")
-@click.argument("output_id")
+@click.option("--output_id", help="Output ID to unflag")
+@click.option("--all", is_flag=True)
```

Comment:
What is `is_flag = True`? You can do `@click.option("--all", type=bool, default-False)`

Reply:
oh ok, I was researching how to do boolean flags with click and this was the option I found that worked best. I didn't know you can set type.

Reply:
Actually! I remember why I did this. this way, if the flag "--all" is passed on the cl, then it is set to True other wise it is False. ie don't have to type --all=True. Easier and more intuitive use

Reply:
awesome